### PR TITLE
Create a Short About Using .goosehints Files with Goose

### LINKED ISSUE #30: Create a Short (YouTube, IG Reel, or TikTok) About Using .goosehints Files with Goose 🦢
# Create a Short Video About Using .goosehints Files with Goose 🦢
[Goose](https://github.com/block/goose) is your on-machine developer agent, automating engineering tasks seamlessly within your IDE or terminal. Guided by you, it can intelligently assess your project's needs, generate code modifications, and implement changes on its own.
Learn more about Goose in our [documentation](https://block.github.io/goose/).

Once your content is merged, it will be featured on our [Community Content site](https://block-open-source.github.io/goose-plugins/)!

## Example clips
https://github.com/user-attachments/assets/0217fc29-3b0b-4ab5-8a60-2dc53eb2b293 

https://github.com/user-attachments/assets/5497112f-7ebb-4e73-9611-191be6d1934f

## 🚀 Goal
Create a vertical short-form video (YouTube Shorts, TikTok, Instagram Reel) demonstrating a quick tip or feature of .goosehints files with Goose. These should be under 60 seconds and in vertical format (9:16 aspect ratio).

## 🤝 How You Can Contribute
Create a quick demo showing:
- Quick setup of .goosehints
- A useful templating tip
- Real project example
- Cool use cases
- Quick tips and tricks
- Time-saving hints setup

## 🔑 Tasks and Acceptance Criteria
- [ ] Create a short video about using .goosehints files
- [ ] Host your content on your preferred platform
- [ ] Add your content to the `content/content.json` file
- [ ] Create a pull request with your changes

Your contribution in content.json must include:
```
{
    "type": "short",
    "thumbnail": "./img/your-thumbnail.png",
    "title": "Your Short Title",
    "link": "https://platform.com/your-short",
    "contributor": "Your Name"
}
```

## 🌟 Resources
- [Goose Documentation](https://block.github.io/goose/)
- [Goose repo](https://github.com/block/goose)
- [.goosehints Documentation](https://block.github.io/goose/guidance/getting-started.html#goosehints)
- [How to use toolkits](https://dev.to/blackgirlbytes/how-my-ai-dev-agent-turns-screenshots-into-ui-updates-189p)
- [How to create toolkits](https://dev.to/blackgirlbytes/how-to-extend-your-ai-agent-for-custom-developer-workflows-4fld)

## Full episodes on Goose to understand more

[How to Build an AI-Powered GitHub Action](https://www.youtube.com/watch?v=lppDgrHY88Y)

[Upgrade U: Supercharging IntelliJ with an Open Source AI Agent](https://www.youtube.com/watch?v=Z4jF4SrbY2c)

## Getting Started
1. Comment ".take" on this issue to get assigned
2. Fork the repository and create a new branch
3. Create your short
4. Submit a PR with your content.json addition
5. Wait for review and address any feedback

## Questions?
If you have any questions or need clarification, please comment on this issue!
Happy creating! 🦢

### PR DESCRIPTION
**Pull request by Lymah123**
Resolves #30

### CODE REVIEW

File: content/content.json [hunk 1]
```diff
@@ -125,6 +125,20 @@
...
       "link": "https://dev.to/lymah/how-to-set-up-goose-on-windows-29dd",
       "contributor": "Lymah123"
+    },
+    {
+      "type": "blog",
+      "thumbnail": "./img/Using .goosehints Files with Goose.png",
+      "title": "Using .goosehints Files with Goose",
+      "link": "https://dev.to/lymah/using-goosehints-files-with-goose-304m",
+      "contributor": "Lymah123"
+    },
```
Comment (blackgirlbytes) [inline content/content.json:135]:
```suggestion
```

Comment (blackgirlbytes) [inline content/content.json:135]:
```suggestion
```

File: content/content.json [hunk 2]
```diff
@@ -125,6 +125,13 @@
       "title": "How to Set Up Goose on Windows",
       "link": "https://dev.to/lymah/how-to-set-up-goose-on-windows-29dd",
       "contributor": "Lymah123"
+    },
+    {
+      "type": "short",
+      "thumbnail": "./img/Short About Using .goosehints Files with Goose.png",
```
Comment (blackgirlbytes) [inline content/content.json:131]: same with this image title..could you please rename to short-about-goosehints-file.png? Thank you!!!

**Review by blackgirlbytes — CHANGES_REQUESTED** [review]

**Comment by Lymah123** [comment]
> Please remove `content/img/Using .goosehints Files with Goose.png` from this PR as it is causing a merge conflict!! Thank you!!

Done! @blackgirlbytes

**Review by blackgirlbytes — APPROVED** [review]
Awesome! thank you for your contribution!!

**Pull request merged**